orttraining/orttraining/python/training/onnxblock/_graph_utils.py

```diff
@@ -183,18 +187,18 @@ def get_model_parameters(model, args_not_requiring_gradient):
     non_trainable_params = []
     for initializer in model.graph.initializer:
         # All model parameters should have their names not begin with
-        # a digit. So, check to see if the initializer's first char
-        # is a digit. If not, it is either a trainable, or a non
+        # `onnx::`. So, check to see if the initializer begins with
```

Comment:
again, is this a "safe" check than the digit?

Reply:
I checked with the exporter team, and this seems to be a safe check. Will ask them if they can provide any utilities that can help us query the initializer names.